src/_shared/scss/_core.scss

```diff
@@ -1,6 +1,6 @@
 $f-serif-headline: 'Guardian Egyptian Web', Georgia, serif !default;
 $f-sans-serif-headline: 'Guardian Text Sans Web', 'Helvetica Neue', Helvetica, Arial, 'Lucida Grande', sans-serif !default;
-$f-serif-text: 'Guardian Text Egyptian Web', Georgia, serif !default;
+$f-serif-text: 'Guardian Egyptian Web', Georgia, serif !default;
```

Comment:
Why this change, I thought the font names were the same? 🤔 

Looks like `Guardian Egyptian Web` got changed to `Guardian Text Egyptian Web` with PR #200 in relation to the Epic work... Do we need separate variables so the different templates all play nice?